macos/FluentUI/Button/Button.swift

```diff
@@ -546,6 +591,18 @@ public enum ButtonStyle: Int, CaseIterable {
 	case borderless
 }
 
+/// Indicates the state of the button for providing different images
+@objc(MSFButtonState)
```

Comment:
This type name will clash with our Vnext Button State if when we bring it to MacOS:
https://github.com/microsoft/fluentui-apple/blob/b2ba3d28873d4d688d14af59de1fd340d312f11e/ios/FluentUI/Vnext/Button/Button.swift#L10

Can you change it to something like **MSFButtonInteractionState**?